docs/schema/validation.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "title": "Validation settings",
+  "markdownDescription": "https://www.mkdocs.org/user-guide/configuration/#validation",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "omitted_files": { "$ref": "#/$defs/omitted_files" },
+    "not_found": { "$ref": "#/$defs/not_found" },
+    "absolute_links": { "$ref": "#/$defs/absolute_links" },
+    "nav": {
+      "title": "validation of navigation",
+      "markdownDescription": "https://www.mkdocs.org/user-guide/configuration/#validation",
+      "additionalProperties": false,
```

Comment:
Please move `"additionalProperties"` below `"properties"`, as in the rest of our schema, e.g., here:

https://github.com/squidfunk/mkdocs-material/blob/57e598b96e8a976eec0559de99ab700bf2d6c38e/docs/schema/plugins/info.json#L36-L46

Reply:
The reason I put it at the top is that it is actually important information for how what follows is to be interpreted.  Happy to follow your style, though. I think the bit about `$defs` is more important.